app/models/notification.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 class Notification < ApplicationRecord
+  include PgSearch
+  pg_search_scope :search_by_subject_title, :against => :subject_title
+
```

Comment:
Also this doesn't by default use a gin index 😬 we would have to create one manually (and with a migration) if we find that the search is too slow. @andrew wdyt